src/components/related-content/_macro.njk

```diff
@@ -1,15 +1,43 @@
 {% macro onsRelatedContent(params) %}
 
+{% from "components/lists/_macro.njk" import onsList %}
+
 {% if params is defined and params and params.classes is defined and params.classes %}
     {% set classes = ' ' + params.classes %}
 {% endif %}
 
   <div class="related-content{{ classes }}">
```

Comment:
I think this could probably be improved using `<aside>` ? 
```suggestion
  <aside class="related-content{{ classes }}" role="complementary">
```
See: https://design-system.service.gov.uk/patterns/start-pages/

Reply:
@jrbarnes9 Good suggestion. I can update this, but this needs to sit outside of `main`, so the main template will need updating to allow for this. Currently the template wraps everything inside `main`